### DISCUSSION TITLE
Can't extract my h4, h5 and display them in sidebar.

### DISCUSSION THREAD

**Discussion by GuillaumeBlochB** [vuepress/core#513] [Q&A] [answered]
Hello,

I've been seeing the PRs announcing a fix had been made for this yet I can't seem to make it work.

My config.js (in docs/.vuepress): 
`module.exports = {
  markdown: {
    extractHeaders: [ 'h2', 'h3', 'h4', 'h5' ]
  }
}`

My page is a very simple Readme.md, no links, only some headings, texts and tables.

Am i forgetting something ?

`"vuepress": "^2.0.0-beta.27"`

Thank you !

Found the solution for those in need : 

I added 
`---
sidebarDepth: 5
---`

at the top of the document

and 
`module.exports = {
  markdown: {
    anchor: {
      level: [2, 3, 4, 5] 
    },
    extractHeaders: {
      level: [2, 3, 4, 5] 
    }
  }
}` from Mister-Hope's answer in the config.js file.

**Comment by Mister-Hope** [comment]
Please read the docs carefully.
https://vuepress2.netlify.app/reference/config.html#markdown-extractheaders

![image](https://user-images.githubusercontent.com/33315834/140604505-b1b66115-e2f1-44e9-a2a3-17c44ad122c5.png)

**Comment by GuillaumeBlochB** [comment]
Hey, yes I've seen this solution, although nothing is getting extracted past 3. 

What is meant by "a subset of markdown.anchor.level" ? Is it something I need to add ?

**Comment by Mister-Hope** [comment]
So this is not working?

```js
module.exports = {
  markdown: {
    extractHeaders: {
      level: [2, 3, 4, 5] 
    }
  }
}
```

**Comment by GuillaumeBlochB** [comment]
Removing 3 did alter the sidebar but adding 4 and 5 did nothing

**Comment by Mister-Hope** [comment, marked as answer]
Try this:

```js
module.exports = {
  markdown: {
    anchor: {
      level: [2, 3, 4, 5] 
    },
    extractHeaders: {
      level: [2, 3, 4, 5] 
    }
  }
}
```

**Comment by GuillaumeBlochB** [comment]
Hey, just got back to work so I can try this. Didn't seem to do anything different. I added a test and h5 isn't extracted. The others are though so that's a start i guess
